doc/source/whatsnew/v0.20.0.txt

```diff
@@ -388,8 +393,7 @@ For example, after running the following, ``styled.xlsx`` renders as below:
    df
    styled = df.style.\
        applymap(lambda val: 'color: %s' % 'red' if val < 0 else 'black').\
```

Comment:
I don't think we have a built-in fucntion for negative values (only for max, min or null)